src/main/resources/init_db.sql

```diff
@@ -0,0 +1,6 @@
+USE jdbc;
+CREATE TABLE books (
+    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    title VARCHAR(255),
+    price DECIMAL
```

Comment:
I'd rather specify the ranges
```suggestion
    price DECIMAL(10,2)
```